.github/workflows/linter.yml

```
@@ -51,5 +51,7 @@ jobs:
         uses: github/super-linter@v4
         env:
           VALIDATE_ALL_CODEBASE: false
+          VALIDATE_JAVASCRIPT_ES: true
+          VALIDATE_MARKDOWN: true
           DEFAULT_BRANCH: main
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Missing newline at the end.

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```